docs/ssl.md

```diff
@@ -20,7 +20,7 @@ The following example enables SSL and automatically generates the certificates:
 
 {{< include-code "enable-ssl.sample" "yaml" >}}
 
-If `sslSecrets.create` is `false`, the operator will look for the secret at `sslSecrets.tlsSecretName` and expect these values:
+If `sslSecrets.create` is `false`, the operator will look for the secret at `sslSecrets.tlsSecretName` in the same namespace of the **KafkaCluster** custom resource and expect these values:
```

Comment:
```suggestion
If `sslSecrets.create` is `false`, the operator will look for the secret at `sslSecrets.tlsSecretName` in the namespace of the **KafkaCluster** custom resource and expect these values:
```

Reply:
Sorry @fekete-robert, I somehow thought I hit the "Commit suggestion" from the UI, but actually I hit the "Resolve conversation" before I merge the PR.

See #117